_includes/ballot-layout.html

```diff
@@ -1,15 +1,18 @@
-{% include election-header.html ballot=include.ballot locality=include.locality %}
+{% assign updated_at = site.data.stats.date_processed | date: "%b. %-d, %Y" %}
 
 <div class="l-ballot">
   <div class="l-ballot__main">
+    <div class="grid-col-12">
+      <span class="l-ballot__updated-at">Content last updated {{ updated_at }}</span>
```

Comment:
It looks like you're adding this in here, but I'm not seeing it removed from anywhere else. How was it on each page before?

Reply:
It used to be in the election-header, which also included the locality. Now locality shows in the ballot-nav instead of the election-header so I eliminated the election-header entirely.
I'm realizing we may ultimately want it back (but maybe under a different name) because the new design has the "currently updated" up there as well as a breadcrumb nav.

Reply:
Ah, got it. Cool, no problem - we can come back to this if/when we need to.